client/src/components/Header/NavBar.js

```diff
@@ -39,6 +41,9 @@ export default function NavBar() {
   const classes = useStyles();
   const preventDefault = (event) => event.preventDefault();
 
+  const user = useUserContext()
+  const { name, photoUrl } = user
+
```

Comment:
can be simplified to:
 const { name, photoUrl } = useUserContext();

Reply:
Good point